dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -289,10 +294,14 @@ const frontsBannerAdStyles = css`
 	position: relative;
 	min-height: ${frontsBannerMinHeight}px;
```

Comment:
Currently when the page loads, we reserve 294px of space for the leaderboard ad, which is too much. So we need to update the reserved height at each breakpoint

```suggestion
	min-height: ${frontsBannerMinHeightTablet}px;
```

Then we'd need to include `min-height: ${frontsBannerMinHeight}px;` at the desktop breakpoint